float64 kff_lamb
---
bool success
string message